README.md

```diff
@@ -40,59 +43,103 @@ Ship is designed to provide first-time configuration UI and/or be used headless
 - Headless mode supports automated pipelines
 - Merge [Helm](https://helm.sh/) charts with override values and apply custom overlays with [Kustomize](https://kustomize.io/) to avoid merge conflicts when upstream or local values are changed
 - Watch upstream repos for updates & sync changes to your local version.
-- Deploy [Helm](https://helm.sh/) charts to a Kubernetes cluster without Tiller
+- Render [Helm](https://helm.sh/) charts to pure Kubernetes yaml that can be deployed to clusters without Tiller
 - Enables [GitOps](https://www.weave.works/blog/the-gitops-pipeline) workflows to update third party applications
 - Configuration workflow `ship.yaml` files can be included in [Helm](https://helm.sh/) chart repos, to customize the initial `ship init` experience
 
-# Installation
-There are two ways you can get started with Ship:
+# Getting Started
```

Comment:
If this isn't going to be the first thing, I think we should consider a table of contents at the top of the readme.  I've used [this tool](https://github.com/ekalinin/github-markdown-toc) for generating a TOC in the past

Reply:
Good call, added a TOC